### DISCUSSION TITLE
thanhle.blog/web3-layer-stack

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#65] [Ideas]
# thanhle.blog/web3-layer-stack

https://thanhle.blog/blog/web3-layer-stack

**Comment by thanhlmm** [comment]
Bài này chủ yếu mình tham khảo ở đây và viết lại dưới góc nhìn của mình https://alchemy.com/blog/web3-stack

Tuy nhiên mình thấy nó còn khá sơ sài và thiếu chiều sâu. Bạn đọc nhớ aware nhé 😅!
